upup/models/cloudup/resources/addons/storage-aws.addons.k8s.io/v1.15.0.yaml.template

```diff
@@ -40,6 +42,22 @@ parameters:
 allowVolumeExpansion: true
 volumeBindingMode: WaitForFirstConsumer
 
+{{ if WithDefaultBool .CloudConfig.AWSEBSCSIDriver.Enabled false }}
+---
```

Comment:
Is there a process for disabling the EBS CSI Driver? I suppose all the PVs using this StorageClass would have to be deleted first or they would become inaccessable. So then it would be safe to delete the StorageClass resource?